DFC.App.JobGroups/Controllers/PagesController.cs

```diff
@@ -48,6 +50,12 @@ public class PagesController : Controller
             {
                 status = "PUBLISHED";
             }
+
+            if (this.configuration != null)
+            {
+                string expiryAppString = this.configuration.GetSection(ExpiryAppSettings).Get<string>();
+                this.expiry = double.Parse(string.IsNullOrEmpty(expiryAppString) ? "4" : expiryAppString);
```

Comment:
Is this expected to fail, if `expiryAppString` is not a number? If so, then this is probably fine. Otherwise, I think some more defensive coding would be good.

I think the reason for using a string when what you want is a double, is if the config value is not a number, you would get a 0. `this.configuration.GetSection(ExpiryAppSettings).Get<double>() // would return 0 if not a number`. However, by using `double.Parse()` you might also be throwing an exception, and it would therefore be better to use `double.TryParse()`.